setup.cfg

```diff
@@ -115,6 +115,8 @@ filterwarnings =
     ignore:::astropy.tests.plugins.display
     ignore:::astropy.tests.disable_internet
     ignore:direct construction of Asdf:pytest.PytestDeprecationWarning
+    ignore:Unknown pytest.mark.mpl_image_compare:pytest.PytestUnknownMarkWarning
```

Comment:
Note: These warnings now appear from `wcsaxes` after replacing `catch_warnings` with `pytest.warns`.

Reply:
Is it worth an issue to fix the tests?

Reply:
I don't think so. This warning is because some tests in `wcsaxes` need the `pytest-mpl` plugin but we cannot install it for all jobs because that would also pull in `matplotlib` and we have jobs that only tests with compulsory dependencies but `matplotlib` is optional. Does that make sense?

Reply:
Yes, makes sense!